src/utils.jl

```diff
@@ -1,6 +1,6 @@
-const _cache = Dict{String,Array}()
+const _cache = Dict{AbstractString,Array}()
```

Comment:
Aaargh... I should not take a shortcut. I'll bite the bullet and set up travis for this now!